powerstore/datasource_volume_group_snapshot.go

```diff
@@ -0,0 +1,258 @@
+						"description": schema.StringAttribute{
+							Description:         "Volume group description.",
+							MarkdownDescription: "Volume group description.",
+							Computed:            true,
+						},
+						"creation_timestamp": schema.StringAttribute{
+							Description:         "The time at which the volume group was created.",
+							MarkdownDescription: "The time at which the volume group was created.",
+							Computed:            true,
+						},
+						"is_protectable": schema.BoolAttribute{
+							Description:         "This is a derived field that is set internally.",
+							MarkdownDescription: "This is a derived field that is set internally.",
+							Computed:            true,
+						},
+						"protection_policy_id": schema.StringAttribute{
+							Description:         "Unique identifier of the protection policy assigned to the volume group.",
+							MarkdownDescription: "Unique identifier of the protection policy assigned to the volume group.",
+							Computed:            true,
+						},
+						"migration_session_id": schema.StringAttribute{
+							Description:         "Unique identifier of the migration session assigned to the volume group when it is part of a migration activity.",
+							MarkdownDescription: "Unique identifier of the migration session assigned to the volume group when it is part of a migration activity.",
+							Computed:            true,
+						},
+						"is_write_order_consistent": schema.BoolAttribute{
+							Description:         "For a primary or a clone volume group, this property determines whether snapshot sets of the group will be write order consistent.",
+							MarkdownDescription: "For a primary or a clone volume group, this property determines whether snapshot sets of the group will be write order consistent.",
+							Computed:            true,
+						},
+						"placement_rule": schema.StringAttribute{
+							Description:         "This is set during creation, and determines resource balancer recommendations.",
+							MarkdownDescription: "This is set during creation, and determines resource balancer recommendations.",
+							Computed:            true,
+						},
+						"type": schema.StringAttribute{
+							Description:         "Type of volume.",
+							MarkdownDescription: "Type of volume.",
+							Computed:            true,
+						},
+						"is_replication_destination": schema.BoolAttribute{
+							Description:         "Indicates whether this volume group is a replication destination.",
+							MarkdownDescription: "Indicates whether this volume group is a replication destination.",
+							Computed:            true,
+						},
+						"protection_data": schema.ObjectAttribute{
+							Description:         "Specifies the ProtectionData associated with a volume group.",
+							MarkdownDescription: "Specifies the ProtectionData associated with a volume group.",
+							Computed:            true,
+							AttributeTypes: map[string]attr.Type{
+								"source_id": types.StringType,
```

Comment:
it would be good to add `expiration_timestamp` attribute as it is associated with volume group snapshot.